packages/kit/src/views/Developer/pages/Gallery/Components/stories/CloudBackupGallery.tsx

```
@@ -16,7 +16,8 @@
~import platformEnv from '@onekeyhq/shared/src/platformEnv';
 import type { IPrimeTransferData } from '@onekeyhq/shared/types/prime/primeTransferTypes';
 
 import { Layout } from './utils/Layout';
+import { showBatchCreateAccountProcessingDialog } from '../../../../../AccountManagerStacks/pages/BatchCreateAccount/ProcessingDialog';
```

ESLint

[error] Too many parent references (5 while maximum of 3 allowed)

(import-path/parent-depth)